frontend/src/metabase/query_builder/containers/QueryBuilder.jsx

```diff
@@ -275,7 +275,7 @@ function QueryBuilder(props) {
   );
 
   const handleSave = useCallback(
-    async (updatedQuestion, { rerunQuery = false } = {}) => {
+    async (updatedQuestion, { rerunQuery } = {}) => {
```

Comment:
Removed the default value to distinguish between `undefined` (for default behavior) and `false` (where we explicitly ask not to run the query)